### PR TITLE
Feature/autotests

### PR DESCRIPTION
**Pull request by alfa24**
#3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,8 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.idea
+/testproject/slave.sqlite3
```
Comment (tumb1er) [inline .gitignore:131]: Let's respect west collegues and use primary-replica terminology.

File: testproject/testapp/tests.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+        Task.objects.create(name='default', project=project)  # w/o exception
+
+    def test_denied_relation__if_from_different_db(self):
+        """ Error relation test if objects are from different DB. """
+        project = Project.objects.create(name='default db', id=1)
+        tag = Tag.objects.create(title='tag db')
+
+        with self.assertRaises(ValueError):
+            project.tags.add(tag)
```
Comment (tumb1er) [inline testproject/testapp/tests.py:115]: I suggest to add some **unit** tests, for all methods in router. 

Also, there should be more tests on allow relation and allow migrate, at least `filter(project__tags__isnull=False)`.

**Review by tumb1er — CHANGES_REQUESTED** [review]
* terminology
* unit tests
* more allow_relation tests

File: .gitignore (hunk 1 above)
Comment (alfa24) [reply .gitignore]: fix

File: testproject/testapp/tests.py (hunk 2 above)
Comment (alfa24) [reply testproject/testapp/tests.py:122]: @tumb1er Please tell me how I can test the allowed migrations, what does this mean?

Comment (tumb1er) [reply testproject/testapp/tests.py:122]: https://docs.djangoproject.com/en/3.2/topics/db/multi-db/#database-routers
There are some methods that the router may implement, and they should be covered by tests also. I mean literally unit tests, when you just instantiate a router and call its methods.

Comment (alfa24) [reply testproject/testapp/tests.py:122]: @tumb1er Check it out, I added some more tests for migrations.

Comment (tumb1er) [reply testproject/testapp/tests.py:122]: :+1:

**Review by tumb1er — APPROVED** [review]

**Pull request merged**